packages/gestalt/src/Module.js

```diff
@@ -1,9 +1,30 @@
 // @flow strict
-import { type Node } from 'react';
+import React, { type Node } from 'react';
+import Box from './Box.js';
+import { type BaseProps } from './moduleTypes.js';
+import { renderModuleTitle } from './moduleUtils.js';
 import ModuleExpandable from './ModuleExpandable.js';
 
-export default function Module(): Node {
-  return null;
```

Comment:
I didn't notice this before, but nice work anticipating this future PR! Makes this a whole lot easier.

Reply:
Not me, all props to @galaxy0101! And I agree, this was really great anticipation